docs/manual.adoc

```diff
@@ -38,27 +36,44 @@ include::ch_installing.adoc[]
 == Running Tali Forth
 include::ch_running.adoc[]
 
-== Major Components
+= Tutorials for Major Components
```

Comment:
Perhaps even just "Introduction to major components"?
It the TOC some of the subsections here have headings like "Tutorial: Working with Blocks" and others don't have the "Tutorial:", like "The Assembler".   I slightly prefer the latter but either way I think they should all be the same.

Reply:
I was trying to mark the ones that were "walkthrough" style as tutorials, where the others are more descriptions of how it works.  Perhaps there is a better way to do this?  or perhaps its not necessary?  
I would like to have a way to find the "hands on things to try" for someone who wants to try things out even if they don't understand everything that is happening.